.travis.yml

```diff
@@ -5,3 +5,4 @@ before_script:
   - npm install -g gulp
 script:
   - npm run build
```

Comment:
I'm wondering if we can avoid building twice on CI? This would significantly increase PR times.

It would be ideal to have the gulp visualtest task just run tests.